### ISSUE TITLE
Error in npm test

### ISSUE THREAD

**Issue by siddhipai** [strongloop/loopback-next#153]
When I `npm test` I get an error in `https://github.com/strongloop/loopback-next/blob/master/packages/loopback/lib/router/SwaggerRouter.ts` which says:

```
Siddhis-MacBook-Pro:loopback-next siddhipai$ npm test

> loopback-next@4.0.0-alpha.1 test /Users/siddhipai/Documents/loopback-next
> mocha --opts test/mocha.opts.root 'packages/**/test/**/*.ts'


/Users/siddhipai/Documents/loopback-next/node_modules/ts-node/src/index.ts:316
          throw new TSError(formatDiagnostics(diagnosticList, cwd, ts, lineOffset))
                ^
TSError: ⨯ Unable to compile TypeScript
packages/loopback/lib/router/SwaggerRouter.ts (210,19): Argument of type 'string' is not assignable to parameter of type 'never'. (2345)
packages/loopback/lib/router/SwaggerRouter.ts (213,19): Argument of type 'string' is not assignable to parameter of type 'never'. (2345)
    at getOutput (/Users/siddhipai/Documents/loopback-next/node_modules/ts-node/src/index.ts:316:17)
    at /Users/siddhipai/Documents/loopback-next/node_modules/ts-node/src/index.ts:347:18
    at Object.compile (/Users/siddhipai/Documents/loopback-next/node_modules/ts-node/src/index.ts:474:19)
    at Module.m._compile (/Users/siddhipai/Documents/loopback-next/node_modules/ts-node/src/index.ts:410:44)
    at Module._extensions..js (module.js:580:10)
    at Object.require.extensions.(anonymous function) [as .ts] (/Users/siddhipai/Documents/loopback-next/node_modules/ts-node/src/index.ts:413:12)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Function.Module._load (/Users/siddhipai/Documents/loopback-next/node_modules/tsconfig-paths/lib/register.js:26:31)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/siddhipai/Documents/loopback-next/packages/loopback/lib/server.ts:10:1)
    at Module._compile (module.js:571:32)
    at Module.m._compile (/Users/siddhipai/Documents/loopback-next/node_modules/ts-node/src/index.ts:410:23)
    at Module._extensions..js (module.js:580:10)
    at Object.require.extensions.(anonymous function) [as .ts] (/Users/siddhipai/Documents/loopback-next/node_modules/ts-node/src/index.ts:413:12)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Function.Module._load (/Users/siddhipai/Documents/loopback-next/node_modules/tsconfig-paths/lib/register.js:26:31)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/siddhipai/Documents/loopback-next/packages/loopback/index.ts:6:1)
    at Module._compile (module.js:571:32)
    at Module.m._compile (/Users/siddhipai/Documents/loopback-next/node_modules/ts-node/src/index.ts:410:23)
    at Module._extensions..js (module.js:580:10)
    at Object.require.extensions.(anonymous function) [as .ts] (/Users/siddhipai/Documents/loopback-next/node_modules/ts-node/src/index.ts:413:12)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Function.Module._load (/Users/siddhipai/Documents/loopback-next/node_modules/tsconfig-paths/lib/register.js:24:35)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/siddhipai/Documents/loopback-next/packages/authentication/test/acceptance/authentication/routing.acceptance.ts:6:1)
    at Module._compile (module.js:571:32)
    at Module.m._compile (/Users/siddhipai/Documents/loopback-next/node_modules/ts-node/src/index.ts:410:23)
    at Module._extensions..js (module.js:580:10)
    at Object.require.extensions.(anonymous function) [as .ts] (/Users/siddhipai/Documents/loopback-next/node_modules/ts-node/src/index.ts:413:12)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Function.Module._load (/Users/siddhipai/Documents/loopback-next/node_modules/tsconfig-paths/lib/register.js:26:31)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at /Users/siddhipai/Documents/loopback-next/node_modules/mocha/lib/mocha.js:222:27
    at Array.forEach (native)
    at Mocha.loadFiles (/Users/siddhipai/Documents/loopback-next/node_modules/mocha/lib/mocha.js:219:14)
    at Mocha.run (/Users/siddhipai/Documents/loopback-next/node_modules/mocha/lib/mocha.js:487:10)
    at Object.<anonymous> (/Users/siddhipai/Documents/loopback-next/node_modules/mocha/bin/_mocha:459:18)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:423:7)
    at startup (bootstrap_node.js:147:9)
    at bootstrap_node.js:538:3
npm ERR! Test failed.  See above for more details.
```

**Comment by rmg** [comment]
This appears to have been introduced in #123.

**Comment by rmg** [comment]
BTW, #123 failed on CI with the same error as seen here.

**Comment by bajtos** [comment]
Fixed by #154
